angular/whereami/src/app/views/navbar/navbar.component.ts

```diff
@@ -30,12 +27,4 @@ export class NavbarComponent implements OnInit {
     this.soundService.complete();
```

Comment:
If this is about line 29:
I would usually prefer the save function over saveForKey, because it is fully self-contained. 
Meaning I actually like the Property objects. 
I only added DefinedPropertyKey, because angular can't resolve constants in html templates so I have to use literals(strings) and type them. 

If this is about line 30(the line you marked): soundeService.complete is called, so a sound is played when you change the setting. Just so the user can gauge how loud it is.

Reply:
It is about line 29.
Thanks for the explanation. Makes sense the way it is now.